src/ReflectionType.php

```diff
@@ -85,14 +85,14 @@ public static function convertToDisplayType(\ReflectionType $type)
             'int'  => 'integer',
             'bool' => 'boolean'
         ];
-        $displayType = $type->type;
+        $displayType = (string)$type;
```

Comment:
👍 , but a test is needed (or post link to existing test), that would make a call to `convertToDisplayType` method with:

1. native `ReflectionType` class object
2. with library's own `ReflectionType` object, that has `type` property